pkg/proxy/nftables/proxier_test.go

```diff
@@ -893,16 +901,16 @@ func TestNodePorts(t *testing.T) {
 			family:            v1.IPv6Protocol,
 			nodePortAddresses: nil,
 
-			allowAltNodeIP: true,
+			allowAltNodeIP: false,
 			expectFirewall: false,
 		},
 		{
 			name: "ipv6, multiple nodeport-addresses",
 
 			family:            v1.IPv6Protocol,
-			nodePortAddresses: []string{"192.168.0.0/24", "192.168.1.0/24", "2001:db8::/64"},
+			nodePortAddresses: []string{"192.168.0.0/24", "192.168.1.0/24", "2001:db8::/64", "2001:db8:1::2/64"},
```

Comment:
```suggestion
			nodePortAddresses: []string{"192.168.0.0/24", "192.168.1.0/24", "2001:db8::/64", "2001:db8:1::2/128"},
```

or `"2001:db8:1::/64"`